src/View/Helper/DatatableHelper.php

```diff
@@ -12,6 +12,8 @@
 use Datatables\Exception\MissConfiguredException;
```

Comment:
write documentation about how to use the feature, add examples and limitations to the feature